frontend/src/api/actions/init.js

```diff
@@ -80,6 +80,9 @@ export function initFromStage(stage, options = { startAfterStage: false }) {
     return async (dispatch, getState) => {
         const parsedGlobals = { mockAPI: null, activeSession: null };
 
+        // These actions do not need to finish in a specific order, so we can wait for them to finish at the end of this function to speed up startup.
+        const asyncActions = [fetchActiveUser, fetchSessions, fetchInstructors];
```

Comment:
Incorrect. `fetchAcitveUser` and `fetchSessions` **must** finish before the others. `fetchInstructors` is the only one that is independent ATM

Reply:
The whole point of an init system is to have things happen in order. Look carefully and where the results of `fetchActiveUser` and `fetchSessions` are used.